cel/canonical_eventlog.go

```diff
@@ -389,5 +394,20 @@ func (c *CEL) replay(bank *pb.PCRs) error {
 		return nil
 	}
 
-	return fmt.Errorf("CEL replay failed for these PCRs in bank %v: %v", tpm2Alg, failedReplayPcrs)
+	return fmt.Errorf("CEL replay failed for these PCRs in bank %v: %v", cryptoHash, failedReplayPcrs)
+}
+
+// VerifyDigest checks the digest generated by the given recornd content to make sure they are equal to
```

Comment:
nit: "generated by the given recornd content to" -> "generated by the given record's content to"

Reply:
Done